release.go

```diff
@@ -0,0 +1,106 @@
+package flux
+
+import (
+	"sort"
+	"time"
+
+	"github.com/weaveworks/flux/guid"
+)
+
+const (
+	PendingReleaseStatus = "pending"
+	RunningReleaseStatus = "running"
+	SuccessReleaseStatus = "success"
+	FailedReleaseStatus  = "failed"
+	SkippedReleaseStatus = "skipped"
+)
+
+type ServiceReleaseStatus string
+
+type ReleaseID string
+
+func NewReleaseID() ReleaseID {
+	return ReleaseID(guid.New())
+}
+
+// Release describes a release
+type Release struct {
+	ID        ReleaseID            `json:"id"`
+	CreatedAt time.Time            `json:"createdAt"`
+	StartedAt time.Time            `json:"startedAt"`
+	EndedAt   time.Time            `json:"endedAt"`
+	Done      bool                 `json:"done"`
+	Priority  int                  `json:"priority"`
+	Status    ServiceReleaseStatus `json:"status"`
```

Comment:
Is this supposed to be an enum (one of those consts above) or a free-form string?